recipes/fastgltf/all/conanfile.py

```diff
@@ -0,0 +1,63 @@
+from conan import ConanFile
+from conan.tools.cmake import CMakeToolchain, CMake, cmake_layout
+from conan.tools.files import get
+from conan.tools.build import check_min_cppstd
+
+class fastgltf(ConanFile):
+    name = "fastgltf"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/spnda/fastgltf"
+    description = "A blazing fast C++17 glTF 2.0 library powered by SIMD."
+    topics = ("gltf", "simd", "cpp17")
+
+    # Needed for CMake configurations for dependencies to work
+    generators = "CMakeDeps"
+
+    settings = "os", "compiler", "build_type", "arch"
+
+    exports_sources = "CMakeLists.txt", "src/*", "cmake/*"
+
+    options = {
+        "fPIC": [True, False],
+        "useSmallVector": [True, False],
+    }
+    default_options = {
+        "fPIC": True,
+        "useSmallVector": False,
+    }
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], strip_root=True)
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def layout(self):
+        cmake_layout(self, src_folder='src')
+
+    def validate(self):
+        check_min_cppstd(self, "17")
```

Comment:
The 1.x pipeline does not user cppstd so it's not actually doing anything at the comment

You can see the FAQ which explains in more details :)

```suggestion
        if self.settings.get_safe("compiler.cppstd"):
            check_min_cppstd(self, "17")
```